eslint/babel-eslint-shared-fixtures/package.json

```diff
@@ -5,21 +5,21 @@
   "license": "MIT",
   "private": true,
   "dependencies": {
-    "@babel/plugin-proposal-class-properties": "^7.1.0",
-    "@babel/plugin-proposal-decorators": "^7.1.2",
-    "@babel/plugin-proposal-do-expressions": "^7.7.4",
-    "@babel/plugin-proposal-nullish-coalescing-operator": "^7.0.0",
-    "@babel/plugin-proposal-optional-chaining": "^7.0.0",
-    "@babel/plugin-proposal-pipeline-operator": "^7.0.0",
-    "@babel/plugin-proposal-private-methods": "^7.7.4",
+    "@babel/plugin-proposal-class-properties": "workspace:^7.1.0",
+    "@babel/plugin-proposal-decorators": "workspace:^7.1.2",
+    "@babel/plugin-proposal-do-expressions": "workspace:^7.7.4",
+    "@babel/plugin-proposal-nullish-coalescing-operator": "workspace:^7.0.0",
+    "@babel/plugin-proposal-optional-chaining": "workspace:^7.0.0",
+    "@babel/plugin-proposal-pipeline-operator": "workspace:^7.0.0",
+    "@babel/plugin-proposal-private-methods": "workspace:^7.7.4",
     "@babel/plugin-syntax-bigint": "^7.7.4",
```

Comment:
Why do syntax plugins not start with the `workspace:` protocol?

Reply:
They are not in this monorepo: they have been moved to `babel-archive`.